content/telegraf/v1/input-plugins/fritzbox/_index.md

```diff
@@ -111,76 +111,86 @@ specification]().
     - `source` - The name of the device (this metric has been queried from)
     - `service` - The service id used to query this metric
   - fields
-    - `layer1_upstream_max_bit_rate` (uint) - The WAN interface's maximum upstream bit rate (bits/sec)
-    - `layer1_downstream_max_bit_rate` (uint) - The WAN interface's maximum downstream bit rate (bits/sec)
-    - `upstream_current_max_speed` (uint) - The WAN interface's current maximum upstream transfer rate (bytes/sec)
-    - `downstream_current_max_speed` (uint) - The WAN interface's current maximum downstream data rate (bytes/sec)
-    - `total_bytes_sent` (uint) - The total number of bytes sent via the WAN interface (bytes)
-    - `total_bytes_received` (uint) - The total number of bytes received via the WAN interface (bytes)
+    - `layer1_upstream_max_bit_rate`   (uint) - WAN interface's maximum upstream
```

Comment:
Extra whitespace added here

Reply:
I'm currently working on getting all READMEs in the Github repo to an 80-column limit. Added the whitespaces here for readability when editing the file in Markdown. Any problem with this?